src/ansys/speos/core/intensity.py

```diff
@@ -224,7 +225,33 @@ def set_FWHM_angle_x(self, value: float = 30) -> Intensity.Gaussian:
             self._gaussian.FWHM_angle_x = value
             return self
 
-        def set_FWHM_angle_y(self, value: float = 30) -> Intensity.Gaussian:
+        def set_FWHM_angle_x(self, value: float = 30) -> Intensity.Gaussian:
+            """Set the full width following x at half maximum.
+
+            .. deprecated:: 0.2.5
```

Comment:
```suggestion
            .. deprecated:: 0.2.2
```
Currently the latest release is 0.2.1 so it should be deprecated in 0.2.2 instead of 0.2.5

Reply:
doesn't mean deprecated removed?
or is there a difference between deprecated and actually removing